i18n/README.ru.md

```diff
@@ -0,0 +1,92 @@
+- [x] Публичная бета: Достаточно стабильна для большинства случаев использования не в Enterprise.
+- [ ] Публичный релиз: Готово к Production использованию.
+
+В настоящее время мы находимся в публичной бете. Следите за разделом "Releases" в этом репозитории, чтобы получать уведомления об основных обновлениях.
+
+<kbd><img src="https://gitcdn.link/repo/supabase/supabase/master/web/static/watch-repo.gif" alt="Watch this repo"/></kbd>
+
+---
+
+## Как это работает
+
+Supabase - это сочетание инструментов с открытым исходным кодом. Мы создаем функционал Firebase, используя инструменты Enterprise уровня с открытым исходным кодом. Если инструмент с открытой лицензией MIT, Apache 2 или аналогичной существует, мы будем использовать и поддерживать его. Если такого инструмента не существует, мы создаем и открываем его в публичный доступ сами. Supabase не является точной копией Firebase. Наша цель - предоставить разработчикам опыт работы с Firebase, используя инструменты с открытым исходным кодом.
+
+**Текущая архитектура**
+
+Supabase – это [облачная платформа](https://app.supabase.io). Вы можете зарегистрироваться и начать использовать Supabase, ничего не устанавливая. Мы продолжаем улучшать удобство локальной разработки - сейчас это наш основной фокус, наряду со стабильностью платформы.
+
+![Архитектура](https://supabase.io/assets/images/supabase-architecture-9050a7317e9ec7efb7807f5194122e48.png)
+
+- [PostgreSQL](https://www.postgresql.org/) – это объектно-реляционная система баз данных с более чем 30 годами активной разработки, которая завоевала прочную репутацию благодаря надежности, устойчивости функционала и производительности.
+- [Realtime](https://github.com/supabase/realtime) – это сервер, написанный на Elixir, который позволяет cледить за вставками, обновлениями и удалениями в базе PostgreSQL с помощью веб-сокета. Supabase использует встроенный функционал репликации Postgres, преобразует поток байтов репликации в JSON, а затем передает JSON через веб-сокеты.
+- [PostgREST](http://postgrest.org/) – это веб-сервер, который превращает вашу базу данных PostgreSQL в RESTful API.
+- [Storage](https://github.com/supabase/storage-api) предоставляет RESTful интерфейс для управления файлами, хранящимися в S3, используя Postgres для управления разрешениями.
+- [postgres-meta](https://github.com/supabase/postgres-meta) – это RESTful API для управления Postgres, позволяющий получать таблицы, добавлять роли, выполнять запросы и т.д.
+- [GoTrue](https://github.com/netlify/gotrue) это API на базе SWT для управления пользователями и выпуска SWT-токенов.
+- [Kong](https://github.com/Kong/kong) это облачный нативный API-шлюз.
+
+#### Клиентские библиотеки
+
+Наша клиентская библиотека является модульной. Каждая подбиблиотека представляет из собя отдельную реализацию для одной внешней системы. Это один из способов, с помощью которого мы поддерживаем существующие инструменты.
+
+- **`supabase-{lang}`**: Объединяет библиотеки и добавляет обогащения.
```

Comment:
In my opinion it's better write like "Объединяет библиотеки и расширяет их функциональность" because original version felt like google translate version